include/common/msgType.h

```diff
@@ -219,7 +221,9 @@ struct handover_cancel_Q_msg {
 struct tauReq_Q_msg {
     int seq_num;
     int enb_fd;
-	int s1ap_enb_ue_id;
+    int s1ap_enb_ue_id;
+    UE_net_capab ue_net_capab;
```

Comment:
these fields may not be mandatory right ? adding bool variable for presence might be good idea

Reply:
Yes, taken care in https://github.com/omec-project/Nucleus/pull/51/commits/d9c789b92c4c58ca8784ef2702f0ec5446ccf359